includes/Context.php

```diff
@@ -384,15 +364,40 @@ private function create_entity_for_post( \WP_Post $post ) {
 	 *
 	 * @since n.e.x.t
 	 *
-	 * @param string $url Input URL.
-	 * @return string URL that starts with the site reference URL.
+	 * @param string $url Optional. Input URL. If not provided, returns the plain reference site URL.
+	 * @return string URL that starts with the reference site URL.
 	 */
-	private function filter_reference_url( $url ) {
-		$reference_site_url = $this->get_reference_site_url();
-		$orig_site_url      = home_url();
+	private function filter_reference_url( $url = '' ) {
+		$orig_site_url = untrailingslashit( home_url() );
```

Comment:
`home_url` is always unslashed, unless someone tampers with it in an `option_` or `home_url` filter. Not a big deal to include here, but shouldn't be necessary.

Reply:
Better safe than sorry, I'd say here. :)